Lib/glyphsLib/classes.py

```diff
@@ -3666,11 +3666,14 @@ def name(self, value):
     def smartComponentPoleMapping(self):
         if "PartSelection" not in self.userData:
             self.userData["PartSelection"] = {}
```

Comment:
do you still need to add "PartSelection" to userData dict in a Font loaded from G3 source where there's already a self.partSelection?